app.test.js

```diff
@@ -253,14 +253,30 @@ describe("GET /api/reviews", () => {
         ]);
       });
   });
-  it("Returns an error when given an category that doesn't exist", () => {
+  it("Returns a 404 status error when given a category that doesn't exist", () => {
     return request(app)
       .get("/api/reviews?category=legacy")
       .expect(404)
       .then(({ body }) => {
         expect(body.message).toBe("This category does not exist");
       });
   });
+  it("Returns a 404 status error when attempting to sort by a key that doesn't exist", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=rating")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("Cannot sort by this input");
+      });
+  });
+  it("Returns a 404 status error when attempting to order by anything other than asc or desc", () => {
+    return request(app)
+      .get("/api/reviews?order=highest")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("Can only order by asc or desc");
+      });
+  });
```

Comment:
I'm on the fence whether or not these should be 400/404. I think there's an argument for both. For what it's worth my notes say 400 as they are now an invalid request as you've greenlisted the ones that work...but I would leave it upto you!